docs/main/concepts/basics.md

```diff
@@ -33,36 +33,36 @@ The [Container Storage Interface (CSI)](https://github.com/container-storage-int
+The dynamic provisioning feature eliminates the need for cluster administrators to pre-provision storage. Instead, it automatically provisions storage when it is requested by users. The implementation of dynamic volume provisioning is based on the `StorageClass` abstraction. A cluster administrator can define as many storage class objects as needed, each specifying a volume plugin (aka provisioner) that provisions a volume and the set of parameters to pass to that provisioner when provisioning. 
 
-A cluster administrator can define and expose multiple flavors of storage (from the same or different storage systems) within a cluster, each with a custom set of parameters. This design also ensures that end users don't have to worry about the complexity and nuances of how storage is provisioned, but still have the ability to select from multiple storage options.
+A cluster administrator can define and expose multiple flavors of storage (from the same or different storage systems) within a cluster, each with a custom set of parameters. This design also ensures that end users do not have to worry about the complexity and nuances of how storage is provisioned, but still have the ability to select from multiple storage options.
 
-When OpenEBS is installed, it ships with a couple of default storage classes that allow users to create either local (OpenEBS Local PV hostPath) or replicated (OpenEBS Jiva) volumes. The cluster administrator can enable the required storage engines and then create Storage Classes for the required Data Engines. 
+When OpenEBS is installed, it ships with a couple of default storage classes that allow users to create either local or replicated volumes. The cluster administrator can enable the required storage engines and then create storage classes for the required Data Engines. 
 
 ## Persistent Volume Claims 
 
-PersistentVolumeClaim (PVC) is a user’s storage request that is served by a StorageClass offered by the cluster administrator. An application running on a container can request a certain type of storage. For example, a container can specify the size of storage it needs or the way it needs to access the data (read only, read/write, read-write many, etc., ).
+Persistent Volume Claim (PVC) is a user’s storage request that is served by a storage class offered by the cluster administrator. An application running on a container can request a certain type of storage. For example, a container can specify the size of storage it needs or the way it needs to access the data (read only, read/write, etc.).
```

Comment:
PersistentVolumeClaim is a single word, StorageClass is also a single word in this sentence.